addons/controllers/clusterbootstrap_controller.go

```diff
@@ -693,6 +693,7 @@ func (r *ClusterBootstrapReconciler) createOrPatchPackageInstallOnRemote(cluster
 
 	_, err = controllerutil.CreateOrPatch(r.context, clusterClient, remotePkgi, func() error {
 		remotePkgi.Spec.ServiceAccountName = r.Config.PkgiServiceAccount
+		remotePkgi.Spec.SyncPeriod = &metav1.Duration{Duration: r.Config.PkgiSyncPeriod}
```

Comment:
Could you please add/update the relevant testcases to check if syncPeriod is set to desired value.

Reply:
Updated and now the tests pass